bql/planner/planner_test.go

```diff
@@ -831,7 +877,7 @@ func TestPlannerQueryError(t *testing.T) {
 		// Actual test:
 		_, err = plnr.Execute(ctx)
 		if err == nil {
-			t.Errorf("planner.Execute(%s)\n= _, nil; want non-nil error", entry.q)
+			t.Errorf("planner.Execute(%s)\n= _, nil; want _, non-nil error", entry.q)
```

Comment:
nit: If you had some error (even if it is a partial message or status) to add to the test cases here it would be better. Given you don't, I guess `error` should be enough (rather than `non-nil error`).

Reply:
Done.